synthesizer/process/src/tests/test_credits.rs

```diff
@@ -1536,9 +1536,9 @@ mod sanity_checks {
         // Compute the assignment.
         let assignment = get_assignment::<_, CurrentAleo>(stack, &private_key, function_name, &[r0, r1, r2], rng);
         assert_eq!(15, assignment.num_public());
-        assert_eq!(54688, assignment.num_private());
-        assert_eq!(54747, assignment.num_constraints());
-        assert_eq!((88514, 130701, 83646), assignment.num_nonzeros());
+        assert_eq!(55698, assignment.num_private());
+        assert_eq!(55761, assignment.num_constraints());
+        assert_eq!((90368, 132045, 84980), assignment.num_nonzeros());
```

Comment:
This pushes us to the next power of two here (2^18, as opposed to 2^17)

Reply:
Great catch, I didn't notice that, thank you!

Reply:
I've addressed the imbalance with this PR: https://github.com/AleoHQ/snarkVM/pull/2087